astroquery/sdss/tests/test_sdss.py

```diff
@@ -318,42 +309,186 @@ def test_query_crossid(patch_post, dr):
     url_tester_crossid(dr)
 
 
+def test_query_crossid_large_radius(patch_request):
+    """Test raising an exception if too large a search radius.
+    """
+    with pytest.raises(ValueError):
```

Comment:
I suggest letting `pytest` [check the error message with the `match` argument](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions) to ensure you are indeed getting the `ValueError` you are expecting. Furthermore, the error message often conveys the purpose of the test clearly enough that an additional comment is not needed.

Reply:
I'll take a look at `match`, but honestly, I'm going to leave the docstring.